From 2b0c2e724124ef8e40f6f494e48606ca5a4124ae Mon Sep 17 00:00:00 2001
From: Ioan Sucan <isucan@gmail.com>
Date: Sat, 15 Feb 2014 20:30:41 -0800
Subject: [PATCH] installation patch

---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1d2f483..d3a73ff 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -55,3 +55,5 @@ install(TARGETS ccd ccd_static ARCHIVE DESTINATION lib${LIB_SUFFIX}/ LIBRARY DES
 install(DIRECTORY src/ccd DESTINATION include FILES_MATCHING PATTERN "*.h")
 install(FILES "${pkg_conf_file}" DESTINATION lib${LIB_SUFFIX}/pkgconfig/ COMPONENT pkgconfig)
 
+install(FILES "${pkg_conf_file}" DESTINATION lib/pkgconfig/ COMPONENT pkgconfig RENAME libccd.pc)
+install(FILES package.xml DESTINATION share/libccd)
-- 
1.9.1.423.g4596e3a

